Description: Use python-config instead of sysconfig.get_python_inc(), as the latter does not work with multiarch-split include files.
Author: Martin Pitt <mpitt@debian.org>
Bug-Ubuntu: https://launchpad.net/bugs/1094246

Index: calibre-1.17.0/setup/extensions.py
===================================================================
--- calibre-1.17.0.orig/setup/extensions.py	2013-12-26 02:14:28.914501680 +0100
+++ calibre-1.17.0/setup/extensions.py	2013-12-26 02:14:28.906501681 +0100
@@ -295,7 +295,11 @@
 if islinux:
     cflags.append('-pthread')
     ldflags.append('-shared')
-    cflags.append('-I'+sysconfig.get_python_inc())
+    import sys
+    config = subprocess.Popen([sys.executable + '-config', '--includes'],
+                              stdout=subprocess.PIPE)
+    cflags += config.communicate()[0].strip().split()
+    assert config.returncode == 0
     ldflags.append('-lpython'+sysconfig.get_python_version())
 
 
